packages/ui-kit/src/components/Logger/Logger.stories.tsx

```diff
@@ -0,0 +1,79 @@
+import React from 'react'
+import type { Meta, StoryObj } from '@storybook/react'
+
+import { storybookCodeTemplate } from '../../helpers'
+import { LogProvider } from './LogProvider'
+import { useLog } from './useLog'
+import { LogLevels } from './Log.types'
+
+const ChildComponent: React.FC<unknown> = () => {
+  const { log, logLevel } = useLog()
+
+  const level = logLevel?.toUpperCase()
+
+  log.error(`This is a ERROR message with the log level set to ${level}`)
```

Comment:
Can you change elsewhere?

```suggestion
  log.error(`This is an error-level message with the log level set to ${level}`)
```